source/_integrations/openhome.markdown

```diff
@@ -10,17 +10,12 @@ ha_codeowners:
   - '@bazwilliams'
 ha_platforms:
   - media_player
+ha_ssdp: true
 ---
 
 The `openhome` platform allows you to connect an [Openhome Compliant Renderer](http://openhome.org/) to Home Assistant such as a [Linn Products Ltd](https://www.linn.co.uk) HiFi streamer. It will allow you to control media playback, volume, source and see the current playing item. Openhome devices should be discovered by using [the discovery component](/integrations/discovery/), their device names are taken from the name of the room configured on the device.
```

Comment:
```suggestion
The Linn / OpenHome integration allows you to connect an [Openhome Compliant Renderer](http://openhome.org/) to Home Assistant such as a [Linn Products Ltd](https://www.linn.co.uk) HiFi streamer. It will allow you to control media playback, volume, source, and see the current playing item. Openhome devices should be discovered by using [the discovery component](/integrations/discovery/), their device names are taken from the name of the room configured on the device.
```

Reply:
Thanks - that reads much nicer!

I've also removed the section about the discovery component which I missed first time around.